cfme/tests/infrastructure/test_tenant_quota.py

```diff
@@ -582,3 +583,134 @@ def test_quota_not_fails_after_vm_reconfigure_disk_remove(request, appliance, fu
 
         # Remove disk UI verification
         assert full_template_vm.configuration.num_disks == orig_config.num_disks
+
+
+@pytest.fixture(scope='module')
+def new_tenants(request, appliance):
+    """This fixture creates two parent tenants"""
+    tenants = []
+    for _ in range(2):
+        tenant = appliance.collections.tenants.create(
+            name=fauxfactory.gen_alphanumeric(18, start="test_parent_"),
+            description=fauxfactory.gen_alphanumeric(18, start="parent_desc_"),
+            parent=appliance.collections.tenants.get_root_tenant()
+        )
+        tenants.append(tenant)
+        request.addfinalizer(tenant.delete_if_exists)
+    return tenants
+
+
+@pytest.fixture(scope='module')
+def new_project(appliance, new_tenants):
+    """This fixture create project under parent tenant1"""
+    tenant1, tenant2 = new_tenants
```

Comment:
If you're not going to use `tenant2`, you could just do just use `_` instead.
`tenant1, _ = new_tenants`